content/en/docs/ops/diagnostic-tools/multicluster/index.md

```diff
@@ -28,6 +28,83 @@ we would expect both `v1` and `v2` responses, indicating traffic is going to bot
 
 There are many possible causes to the problem:
 
+### Connectivity and firewall issues
+
+In some environments it may not be apparent that a firewall is blocking traffic between your clusters. It's possible
+that `ICMP` (ping) traffic may succeed, but HTTP and other types of traffic do not. This can appear as a timeout, or 
+in some cases a more confusing error such as:
+
+{{< text >}}
+upstream connect error or disconnect/reset before headers. reset reason: local reset, transport failure reason: TLS error: 268435612:SSL routines:OPENSSL_internal:HTTP_REQUEST
+{{< /text >}}
+
+While Istio provides service discovery capabilities to make it easier, cross-cluster traffic should still succeed
+if pods in each cluster are on a single network without Istio. To rule out issues with TLS/mTLS, we can do a manual
+traffic test on non-sidecared pods.
```

Comment:
nit: I am not a docs writer but seems like `sidecared` may not be the right phrase here. Not sure what is better though - pods without a sidecar proxy?

Reply:
+1